source/extensions/priv/server/ntds_jet.c

```diff
@@ -0,0 +1,312 @@
+#include "precomp.h"
+
+JET_ERR engine_shutdown(jetState *ntdsState){

```

Comment:
Given that the shutdown function does some cleaning up of memory that something else has allocated, perhaps we should add some Doxygen documentation to the functions so that it's clear?